test/sqllogictest/filter-pushdown.slt

```diff
@@ -140,3 +140,32 @@ Source materialize.public.events
   pushdown=(#1, #2)
 
 EOF
+
+# Verify explain behaviour for functions with many arguments. In theory, we can't push down
+# non-associative functions with long argument lists... but in practice all the functions we
+# can push down are also associative, so this is moot. Let's at least check that an associative
+# function _does_ report pushdown even when the argument list is long.
```

Comment:
nit: Wasn't immediately obvious to me that the variadic thing here is the `AND`.

Reply:
Just saw you merged. Definitely not worth changing anything here!

Reply:
I'll roll it up with some other documentation changes!